mozilla-release/mobile/android/app/src/main/res/layout/home_myoffrz_panel.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Add the following as first line
```xml
<!-- Copyright © Cliqz 2018 -->
```
Just to mark the files we add as belonging to Cliqz

Reply:
done, also for other xml files